rpcs3/Emu/RSX/VK/VKGSRender.cpp

```diff
@@ -1698,7 +1697,7 @@ void VKGSRender::do_local_task(rsx::FIFO_state state)
 	{
 	case rsx::FIFO_state::lock_wait:
 		// Critical check finished
-		return;
+		break;
```

Comment:
Same as above. Don't flip when waiting for semaphore to release. We can do better scheduling than that.

Reply:
Updated.

Reply:
Thanks.